Doc/library/exceptions.rst

```diff
@@ -921,6 +930,21 @@ their subgroups based on the types of the contained exceptions.
       >>> MyGroup("eg", [ValueError(1), TypeError(2)]).split(TypeError)
       (MyGroup('eg', [TypeError(2)]), MyGroup('eg', [ValueError(1)]))
 
+   Note that :exc:`BaseExceptionGroup` defines :meth:`__new__`, so
```

Comment:
This block is in the docs for the `derive` method, but I think it makes more sense in the top-level description of the class (here: https://docs.python.org/3.11/library/exceptions.html#BaseExceptionGroup).

Also, I noticed the descriptions of the methods are underindented in the docs. They should be indented by one more level, as you did for the attributes above.

Reply:
I agree this belong at a higher level. On the other hand I didn't want to present it in the beginning before we even mentioned derive(). Would it work if I indent everything as you suggest, and leave the full subclassing example at its current indentation level?

Reply:
Yes, good idea